devel/libjob_queue/src/job_queue.c

```diff
@@ -1312,6 +1314,7 @@ void job_queue_set_max_running( job_queue_type * queue , int max_running ) {
 /*
   The return value is the new value for max_running.
 */
+/// TODO Delete this function? (note: 1, not used. 2, spelled runnning)
 int job_queue_inc_max_runnning( job_queue_type * queue, int delta ) {
   job_queue_set_max_running( queue , job_queue_get_max_running( queue ) + delta );
   return job_queue_get_max_running( queue );
```

Comment:
Simply a question whether I can delete this function which is unused (as far as I can tell.)